fit/README.md

```diff
@@ -0,0 +1,61 @@
+# Fit file as available from Strava
+
+Pull your own data from your Strava account. Visualize your data, manipulate your data, combine your data with other sources,  all while maintaining full ownership and control of the data in your own environment.
+
+## Table of contents
+
+ * `ThursMorn.fit` - the data used to run this example
+ * `accessFit.py` - python script to demonstrate fit data inside deephaven 
+ * `*.png` - images used in this help
+
+## Steps to run: example or your own data
+
+1. Login to your Strava account
+1. Select an activity.
```

Comment:
these are all number 1

Reply:
Intentionally so... I thought that was the way to make markdown auto-number them.. so if you go add another step in the middle, all the steps after that still have the correct numbering.

If we don't like doing this stylistically, I'm happy to change it?